go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/karpenter
 
-go 1.23.0
```

Comment:
Do you know why these vulnerabilities don't show up in `govulncheck`. Also, seems like other things were generated with this change from `make presubmit` that you need to include in this PR

Reply:
@jonathan-innis this might be related to https://github.com/golang/go/issues/51653